### PR TITLE
Caching Payment Processors Endpoint Output

### PR DESCRIPTION
**Pull request by clintonb**
This information will not change frequently.

@jimabramson @rlucioni

### CODE REVIEW

File: ecommerce/extensions/api/v2/urls.py [hunk 1]
```diff
@@ -33,7 +34,7 @@
 
 PAYMENT_URLS = patterns(
     '',
-    url(r'^processors/$', views.PaymentProcessorListView.as_view(), name='list_processors'),
+    url(r'^processors/$', cache_page(1800)(views.PaymentProcessorListView.as_view()), name='list_processors'),
```
Comment (rlucioni) [inline ecommerce/extensions/api/v2/urls.py]: I think it'd make sense for the cache timeout to be specified in settings. I'd also specify the timeout as `60 * 30` for the sake of readability.

**Comment by jimabramson** [comment]
@clintonb good call.  In fact this data will _never_ change (during runtime) so it seems like a profoundly long timeout would be fine, but at the same time, the view is pretty cheap to begin with.

**Comment by jimabramson** [comment]
:+1:

File: ecommerce/extensions/api/v2/urls.py (hunk 1 above)
Comment (clintonb) [inline ecommerce/extensions/api/v2/urls.py]: Updated for readability. I don't think this value warrants its own setting at this point. It's not really worth configuring.

**Comment by rlucioni** [comment]
:+1:

**Pull request merged**